### PR TITLE
image upload

### PR DESCRIPTION
**Pull request by MathiasHobler**

### CODE REVIEW

File: src/components/Inputfield/ImageInput.jsx [hunk 1]
```diff
@@ -1,88 +1,56 @@
...
+		setImageURL({
+			data: imageURL.data,
+			error: false,
+			loading: true,
+		});
+		await axios({
+			method: 'post',
+			url: 'https://api.imgur.com/3/image',
+			headers: {
+				Authorization: `Client-ID fc88c8f5835b3ec`,
```
Comment (macfrei) [inline src/components/Inputfield/ImageInput.jsx:28]: Is this not supposed to be a secret? 🤔

**Review by macfrei — COMMENTED** [review]
Just one minor comment :)

File: api/imgur/upload.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+export default async function handler(request, response) {
+	const form = formidable({});
+	form.parse(request, async (error, fields, files) => {
+		if (error) {
+			response.status(400).json({message: error.message});
+		} else {
+			const {file} = files;
+			const {newFilename, filepath} = file;
+			console.log(request.body);
```
Comment (stefankorder) [inline api/imgur/upload.js:24]: Please delete this console log.

File: src/components/Inputfield/ImageInput.jsx [hunk 3]
```diff
@@ -1,88 +1,81 @@
...
+				},
+			});
+			setImageURL({
+				data: data.url,
+				success: true,
+				error: false,
+				loading: false,
+			});
+			setNewEvent({...newEvent, eventPicture: data.url});
+			console.log(data.url);
```
Comment (stefankorder) [inline src/components/Inputfield/ImageInput.jsx:43]: Please delete this console los.

**Review by stefankorder — CHANGES_REQUESTED** [review]
Good work! There are just two unneeded console logs in your code.

File: api/imgur/upload.js (hunk 2 above)
Comment (MathiasHobler) [reply api/imgur/upload.js]: Done

**Review by stefankorder — APPROVED** [review]
Good work!

**Pull request merged**